### PR TITLE
Arn 422 extract data from redis

### PR DESCRIPTION
**Pull request by adelaidaa**
Those are the changes to integrate reading the OasysUserCode from the prepopulated Redis cache from the `hmpps-assessments-ui`

### CODE REVIEW

File: helm_deploy/hmpps-assessments-api/templates/_envs.tpl [hunk 1]
```diff
@@ -109,4 +109,19 @@ env:
...
+        name: {{ .Values.redis.secret }}
+        key: primary_endpoint_address
+
+  - name: REDIS_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: {{ .Values.redis.secret }}
+        key: auth_token
+
+  - name: REDIS_TLS_ENABLED
```
Comment (LucasCairns) [inline helm_deploy/hmpps-assessments-api/templates/_envs.tpl:124]: Looking at the `application.properties` should this environment variable be called `REDIS_SSL`?

File: src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceOASysTest.kt [hunk 2]
```diff
@@ -312,7 +312,7 @@ class AssessmentUpdateServiceOASysTest {
         )
       )
     )
-    every { assessmentUpdateRestClient.updateAssessment(any(), any(), any(), any(), any()) } returns oasysError
+    every { assessmentUpdateRestClient.updateAssessment(any(), any(), any(), any()) } returns oasysError
     // Christ, what a lot of set up
```
Comment (LucasCairns) [inline src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceOASysTest.kt:316]: 😂

File: helm_deploy/hmpps-assessments-api/templates/_envs.tpl (hunk 1 above)
Comment (LucasCairns) [reply helm_deploy/hmpps-assessments-api/templates/_envs.tpl]: Pushed fix

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

**Review by SWhitlam — APPROVED** [review]

File: helm_deploy/hmpps-assessments-api/templates/_envs.tpl (hunk 1 above)
Comment (adelaidaa) [reply helm_deploy/hmpps-assessments-api/templates/_envs.tpl]: Awesome, thanks @LucasCairns

File: src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceOASysTest.kt (hunk 2 above)
Comment (adelaidaa) [reply src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceOASysTest.kt:316]: the perfect test, naughty devs!!!

**Pull request merged**